helmfile/01-applications.yaml

```diff
@@ -112,6 +112,7 @@ releases:
 {{ end }}
 {{ end }}
 
+{{ if .Values.velero.enabled }}
```

Comment:
What happens if you have installed it with `velero.enabled=true` and then set it to `false` and run apply again?
Does it uninstall the chart or does nothing happen? 

I know that if you use `condition` for the charts it will just not do anything if you go from enabled to disabled

Reply:
Tried it now and nothing happens if i set `enabled=false` on a cluster that already have velero installed. It would be nice if velero was uninstalled by setting it to false. But I'm not sure how to make that happen.

Reply:
I guess you'd need to add `{{ if .Values.velero.enabled }}` around all templates in the chart. But that's not an option for us to do that for all charts. So if there's no other smart way, I would say that this is good as it is.
But then I think you can change the if statement to just a `condition` on the chart instead. If you think that's better looking.

Reply:
I think the way to do this is to set `installed: false` or use the `condition` field like @Xartos mentioned.
Example:
```yaml
releases:
- name: velero
  namespace: velero
  # Not sure if condition will work
  condition: velero.enabled
  # If it doesn't this should!
  installed: false
```

Reply:
`installed: false` seems to work great. It deletes the chart if it was present before and will not complain if some object storage values are not set. Will provide an update soon!